src/_includes/footer.html

```diff
@@ -1,18 +1,13 @@
 <footer class="site-footer">
-    <div class="site-footer-container">
-  
-        <div class="container">
-        
-            <div class="footer-col-1">
-                <p><strong>TestCafe</strong>: Web Testing Framework<br/>
-                Copyright © 1998-{{ site.time | date: "%Y" }} Developer Express Inc.</p>
-            </div>
+    <div class="site-footer-content">
+        <div class="footer-col-1">
+            <p><strong>TestCafe</strong>: Web Testing Framework<br/>
+            Copyright © 1998-{{ site.time | date: "%Y" }} Developer Express Inc.</p>
+        </div>
 
-            <div class="footer-col-2">
-                <p>TestCafe is the easiest way to automate QA processes and test your<br/>
-                websites and apps across browsers, operating systems and devices.</p>
-            </div>
-            
+        <div class="footer-col-2">
+            <p>TestCafe is the easiest way to automate QA processes and test your<br/>
+            websites and apps across browsers, operating systems and devices.</p>
```

Comment:
This is from the marketing part of the current site. It must be Julian's phrase (or, at least, Abadzhev's).
Since what you bring up is more a style issue, I won't dare to override these people =)